worldeditadditions_core/register/check.lua

```diff
@@ -0,0 +1,14 @@
+function worldeditadditions_core.register_command(def)
+	local def = table.copy(def)
```

Comment:
I can't find `table.copy` in the official Lua docs? Might be best to use `wea.table.shallowcopy`?

Reply:
Did you check the [minetest api](https://github.com/minetest/minetest/blob/master/doc/lua_api.txt#L3370)?

Reply:
Oh neat, I didn't realise it's Minetest specific

I would get somewhat nervous about using that in a backend backend API function (e.g. Vector3), but this depends on Minetest already so it doesn't matter much. Let's keep this as-is.